src/node.cc

```diff
@@ -3687,6 +3683,15 @@ void Init(int* argc,
     }
   }
 
+#ifdef __POSIX__
+  // Block SIGPROF signals when sleeping in epoll_wait/kevent/etc.  Avoids the
+  // performance penalty of frequent EINTR wakeups when the profiler is running.
+  // Only for v8.log profiling. CpuProfiler API expects to have evenly distributed samples.
```

Comment:
Not sure about that. SIGPROF is most likely not defined on Windows, and specific numerical value is not portable.